pinakes/main/migrations/0038_seed_email_notification.py

```diff
@@ -0,0 +1,132 @@
+                "type": "string",
+                "validate": [
+                    {"type": "required-validator"},
+                ],
+            },
+            {
+                "label": "Port",
+                "name": "port",
+                "initialValue": 25,
+                "helper_text": "Port to use for the SMPT server",
+                "component": "text-field",
+                "type": "string",
+                "dataType": "integer",
+                "validate": [
+                    {"type": "required-validator"},
+                    {"type": "min-number-value", "value": 1},
+                ],
+            },
+            {
+                "label": "Username",
+                "name": "username",
+                "helper_text": "Username to authenticate with the SMTP server",
+                "isRequired": False,
+                "component": "text-field",
+                "type": "string",
+            },
+            {
+                "label": "Password",
+                "name": "password",
+                "helper_text": "Password to authenticate with the SMTP server",
+                "isRequired": False,
+                "component": "text-field",
+                "type": "password",
+            },
+            {
+                "label": "Security",
+                "name": "security",
+                "helper_text": "Choose a security connection method to SMTP",
+                "component": "select-field",
+                "isRequired": False,
+                "options": [
+                    {"label": "SSL", "value": "use_ssl"},
+                    {"label": "TLS", "value": "use_tls"},
+                ],
+            },
+            {
+                "label": "SSL Keyfile",
+                "name": "ssl_keyfile",
+                "helper_text": (
+                    "Path to a PEM-formatted private key file to use for the "
+                    "SSL connection"
+                ),
+                "isRequired": False,
+                "component": "text-field",
+                "type": "string",
+            },
+            {
+                "label": "SSL Certfile",
+                "name": "ssl_certfile",
+                "helper_text": (
+                    "Path to a PEM-formatted certificate chain file to use for"
+                    " the SSL connection"
+                ),
+                "isRequired": False,
+                "component": "text-field",
+                "type": "string",
+            },
+            {
+                "label": "From",
+                "name": "from",
+                "helper_text": "Sender's email address",
+                "isRequired": True,
+                "component": "text-field",
+                "type": "string",
+                "validate": [
+                    {"type": "required-validator"},
+                    {
+                        "type": "pattern-validator",
+                        "pattern": "[a-z0-9._%+-]+@[a-z0-9.-]+.[a-z]{2,}$",
```

Comment:
Not sure whether we want to validate the email address. According to https://stackoverflow.com/questions/46155/whats-the-best-way-to-validate-an-email-address-in-javascript there is no perfect regex for email validation.
In addition the from field can be something like `First Last <name@mail.com>` which will be rejected by such validation.